DesignSafe-CI/portal
# Task/WG 370 open topo UI improvements

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [WG-370](https://tacc-main.atlassian.net/browse/WG-370)

## Summary of Changes: ##


## Testing Steps: ##
1. Go to https://designsafe.dev/recon-portal/
2. Click 'Show filter options' above the map
3. Click View External Dataset Filters
4. Make sure listings and event markers appear/hide as is appropriate 

## UI Photos:
<img width="1785" alt="Screenshot 2024-08-27 at 7 16 50 PM" src="https://github.com/user-attachments/assets/47cd7180-237f-409b-9586-e69ccb2c03b2">
<img width="1784" alt="Screenshot 2024-08-27 at 7 16 35 PM" src="https://github.com/user-attachments/assets/54f06d34-ff03-4cda-a71e-5458a55dacb2">



## Notes: ##
External Data 2 will only mimic the open topo filter. Implementation will need to come later - just put this placeholder for CMD to work with and keep in mind. Can remove before merging.
